src/lib/validate.ts

```diff
@@ -49,11 +49,11 @@ export function compare(oldSwagger: string, newSwagger: string, options: OpenApi
 export function compareTags(
   oldSwagger: string, oldTag: string, newSwagger: string, newTag: string, options: OpenApiDiff.Options
 ) {
-  if (!options) options = {};
+  if (!options) options = {}
 
-  log.consoleLogLevel = options.consoleLogLevel || log.consoleLogLevel;
-  log.filepath = options.logFilepath || log.filepath;
-  let openApiDiff = new OpenApiDiff.OpenApiDiff(options);
+  log.consoleLogLevel = options.consoleLogLevel || log.consoleLogLevel
+  log.filepath = options.logFilepath || log.filepath
+  let openApiDiff = new OpenApiDiff.OpenApiDiff(options)
 
-  return openApiDiff.compare(oldSwagger, newSwagger, oldTag, newTag);
-};
+  return openApiDiff.compare(oldSwagger, newSwagger, oldTag, newTag)
```

Comment:
Let's double check on this comparison, which was what @sarangan12 was referring to regarding the different inputs. Let's be consistent on the compare output for both functions, compare and compareTags

Reply:
A new test has been added.